internal/configuration/config.go

```diff
@@ -44,6 +46,17 @@ type Config struct {
 	Daemon bool `yaml:"-"`
 }
 
+type EventsConfig struct {
+	Enabled bool                   `yaml:"enabled"`
+	Sinks   []*WavefrontSinkConfig `yaml:"sinks"`
```

Comment:
This will go away. We'll have a single sink that receives metrics / events and sends it to Wavefront.